doc/api/esm.md

```diff
@@ -1232,13 +1236,19 @@ The resolver has the following properties:
 * No default extensions
 * No folder mains
 * Bare specifier package resolution lookup through node\_modules
+* Does not fail on unknown extensions or protocols
```

Comment:
Perhaps we can explicitly note the load phase:

```
When loading, a resolved URL, the default Node.js loader will:
* Throw for unknown extensions / module formats.
* Throw for unsupported schemes.

Loaders are able to customize the resolution behaviour further.
```

Reply:
Something like this is what I had in mind. Like, rename the “Node.js resolution algorithm” header to “Node.js resolution and loading algorithm” and add a few bullet points for the loading phase: each of the errors that could be thrown, and then “The module is loaded from disk or network.” That’s it.